infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/utils/InfinispanRemoteTestHelper.java

```diff
@@ -143,6 +171,50 @@ public long getNumberOfEntities(SessionFactory sessionFactory) {
 
 	// Various static helpers below:
 
+	public static void registerScriptStoredProcedures(SessionFactory sessionFactory) {
```

Comment:
@ntviet18 what you think about the idea of moving new helper methods in a dedicated helper class? It is not critical, it is just to walk about it :P

Reply:
sure, I think it's a good idea. I will move it